.github/workflows/build.yml

```
@@ -155,3 +155,160 @@ jobs:
                     name: Behat logs
                     path: etc/build/
                     if-no-files-found: ignore
+
+    tests-with-pdf-generation-disabled:
+        runs-on: ubuntu-latest
+
+        name: "PDF generation disabled: Sylius ${{ matrix.sylius }}, PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }}, MySQL ${{ matrix.mysql }}"
+
+        strategy:
+            fail-fast: false
+            matrix:
+                php: ["8.1"]
+                symfony: ["^5.4"]
+                sylius: ["~1.11.0"]
+                node: ["14.x"]
+                mysql: ["8.0"]
+
+        env:
+            APP_ENV: test
+            DATABASE_URL: "mysql://root:root@127.0.0.1/sylius?serverVersion=${{ matrix.mysql }}"
+
+        steps:
+            -
+                uses: actions/checkout@v2
+
+            -
+                name: Setup PHP
+                uses: shivammathur/setup-php@v2
+                with:
+                    php-version: "${{ matrix.php }}"
+                    extensions: intl
+                    tools: symfony
+                    coverage: none
+
+            -
+                name: Setup Node
+                uses: actions/setup-node@v1
+                with:
+                    node-version: "${{ matrix.node }}"
+
+            -
+                name: Shutdown default MySQL
+                run: sudo service mysql stop
+
+            -
+                name: Setup MySQL
+                uses: mirromutth/mysql-action@v1.1
+                with:
+                    mysql version: "${{ matrix.mysql }}"
+                    mysql root password: "root"
+
+            -
+                name: Remove wkhtmltopdf binary
+                run: rm tests/Application/etc/wkhtmltopdf
+
+            -
+                name: Disable PDF generation
+                run: mv tests/Application/etc/sylius_invoicing_pdf_generation_disabled.yaml tests/Application/config/packages
```

if only these two steps are different perhaps we can plug this build again to the the main one with included following values:
```
                php: "8.1"
                symfony: "^5.4"
                sylius: "~1.11.0"
                node: "14.x"
                mysql: "8.0"
                pdf: false
```

and the add 
```suggestion
            -
                name: Remove wkhtmltopdf binary
                if: matrix.pdf != ''
                run: rm tests/Application/etc/wkhtmltopdf

            -
                name: Disable PDF generation
                if: matrix.pdf != ''
                run: mv tests/Application/etc/sylius_invoicing_pdf_generation_disabled.yaml tests/Application/config/packages
```